packages/domain/src/use-cases/publications/CreateQuote.ts

```diff
@@ -52,11 +34,7 @@ export class CreateQuote extends SponsorshipReady<CreateQuoteRequest> {
   }
 
   protected override async sponsored(request: CreateQuoteRequest): Promise<void> {
-    if (
-      request.reference.type === ReferencePolicyType.ANYONE &&
-      request.actions.length === 0 &&
-      isMomokaPublicationId(request.quoteOn)
-    ) {
```

Comment:
I don't know what I was thinking, just the `quoteOn` is relevant for making a decision on Momoka or on-chain. Same applies to mirrors and comments.